File: app/src/main/java/com/akexorcist/screenshotdetection/sample/MainActivity.java
package com.akexorcist.screenshotdetection.sample;

import android.os.Bundle;
import android.support.design.widget.Snackbar;
import android.view.View;

public class MainActivity extends ScreenshotDetectionActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    @Override
    public void onScreenCaptured(String path) {
        Snackbar.make(getRootView(), path, Snackbar.LENGTH_SHORT).show();
    }

    @Override
    public void onScreenCapturedWithDeniedPermission() {
        Snackbar.make(getRootView(), R.string.please_grant_read_external_storage_permission_for_screenshot_detection, Snackbar.LENGTH_SHORT).show();
    }

    private View getRootView() {
        return getWindow().getDecorView().findViewById(android.R.id.content);
    }
}

File: app/src/main/java/com/akexorcist/screenshotdetection/sample/MainActivity.kt
package com.akexorcist.screenshotdetection.sample

import android.os.Bundle
import com.akexorcist.screenshotdetection.sample.databinding.ActivityMainBinding
import com.google.android.material.snackbar.Snackbar

class MainActivity : ScreenshotDetectionActivity() {
    private val binding: ActivityMainBinding by lazy { ActivityMainBinding.inflate(layoutInflater) }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(binding.root)
    }

    override fun onScreenCaptured(path: String) {
        Snackbar.make(binding.root, path, Snackbar.LENGTH_SHORT).show()
    }
}
